go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/minio/minio
 
 go 1.19
 
+replace github.com/minio/console v0.28.0 => github.com/kaankabalak/console v0.3.27-0.20230519061458-4cb5186da8d1
```

Comment:
As required PRs merged, please correct these entries with proper ones.

Reply:
I will update this file as soon as we make a Console release 👍

Reply:
Done, I have updated the dependencies. The PR can now be merged after approving reviews 👍